scripts/test-node-fetch.js

```diff
@@ -0,0 +1,11 @@
+const [major, minor, patch] = process.versions.node.split('.').map(v => Number(v))
+const required = process.argv.pop().split('.').map(v => Number(v))
+
+const badMajor = major < required[0]
+const badMinor = major === required[0] && minor < required[1]
+const badPatch = major === required[0] && minor < required[1] && patch < required[2]
```

Comment:
I did a little mistake here by accident, it should be `===` in the middle condition 😅